src/rootPages/Designer/properties/process/ABProcessParticipant_selectManagersUI.js

```diff
@@ -7,27 +7,22 @@
  * @return {ClassUI} The Class Definition for this UI widget.
  */
 import UI_Class from "../../ui_class";
+import Participant_selectManagersUI from "./Participant_selectManagersUI";
 
 export default function (AB) {
    const UIClass = UI_Class(AB);
    var L = UIClass.L();
 
-   class UIProcessParticipant_selectManagersUI extends UIClass {
+   class UIProcessParticipant_selectManagersUI extends Participant_selectManagersUI {
       constructor(id) {
-         super(id, {
-            form: "",
-            name: "",
-            role: "",
-            useRole: "",
-            useAccount: "",
-            useField: "", // bool on whether to use userFields from process
-            account: "",
-            fields: "", // to\fromUsers.fields
-            userField: "",
-         });
+         super(id);
       }
 
       ui(obj = {}) {
+         const baseUI = super.ui(obj);
+
+         baseUI.elements.push({});
```

Comment:
You should push the webix config for the `userField`  row. (Starts on line 224).

Reply:
Also don't need the `__Roles` and  `__Users` below. That should be handled by the parent class.
And `return` the `baseUI` instead of repeating the same code.

Reply:
Thank You Nathan